File: GymnasticsStudioServer/Dal/Student.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace Dal
{
    public class Student
    {
        public int Id;
        public string FirstName;
        public string LastName;
        public string IdentityNumber;
        public string PhoneNumber;
        public string MobilePhoneNumber;
        public int Balance=87;


        public Student()
        { }

        public Student(int id ,string firstName ,string lastName ,string identityNumber,string phoneNumber,string mobilePhoneNumber,int balance)
       {
            this.Id = id;
            this.FirstName = firstName;
            this.LastName = lastName;
            this.IdentityNumber = identityNumber;
            this.PhoneNumber = phoneNumber;
            this.MobilePhoneNumber = mobilePhoneNumber;
            this.Balance = balance;
        }
    }



}

File: GymnasticsStudioClient/src/app/classes/student.ts

export class Student {
    Id: number
    FirstName: string
    LastName: string
    IdentityNumber: string
    PhoneNumber: string
    Pignicher: string
    StudentKind: string
    Balance: number = 0
    CreditDetailsId: number = 0
    Addrees: string
    BirthDay: Date
    HMO: string
    Comments: string
    StartDate: string
    CreditCardNumber: string
    CreditCardValidity: Date
    CreditCardBackDigitis: string
    constructor(init?: Partial<Student>) {
        Object.assign(this, init);
    }
}
